tests/test_e2e_05_topology.py

```diff
@@ -31,10 +31,26 @@ def setup_class(cls):
     def teardown_class(cls):
         cls.net.stop()
 
-    def test_010_list_switches(self):
+    def test_005_list_topology(self):
         """
         Test /api/kytos/topology/v3/ on GET
         """
+        api_url = KYTOS_API + '/topology/v3/'
+        response = requests.get(api_url)
```

Comment:
When we request the API /topology/v3/ it will return switches, links and interfaces. The test is only validating switches. It would be nice if the test also validates the interfaces and links.